index_BCCWJ.html

```diff
@@ -9,10 +9,10 @@
   <meta property="og:image:type" content="image/png"/>
   <meta property="og:image:width" content="256"/>
   <meta property="og:image:height" content="256"/>
-  <meta property="og:title" content="Anki Frequency Inserter | BCCWJ version"/>
+  <meta property="og:title" content="Anki Frequency Inserter | Anime version"/>
   <meta property="og:type" content="website"/>
   <meta property="og:updated_time" content="1631042980"/>
-  <title>Anki Frequency Inserter (BCCWJ version)</title>
+  <title>Anki Frequency Inserter (Anime version)</title>
```

Comment:
wrong file (this is _BCCWJ.html). you already added this in _anime.html, so this change should just be reverted I think.